website/versioned_docs/version-v3.9.x/expansion.md

```diff
@@ -0,0 +1,218 @@
+---
+id: expansion title: Expansion
```

Comment:
i don't believe id can have spaces (this is the part that gets referenced in sidebars), we'll need a title too

@maxsmythe @ritazh  what do we want to call this? "Template Extension", "Workload Expansion"?

Reply:
"Early Rejection of Generator Resources"?

A bit jargony, but at least is focused on what users will mainly use this for.

Reply:
I also like "Early Rejection of Generator Resources", as it makes it clear what the feature does just from the title.